src/test/resources/exception-diff-sample_json-path.txt

```diff
@@ -0,0 +1,16 @@
+
+                                               Request was not matched
+                                               =======================
+
+-----------------------------------------------------------------------------------------------------------------------
+| Closest stub                                             | Request                                                  |
+-----------------------------------------------------------------------------------------------------------------------
+                                                           |
+ANY                                                        | ANY
+/thing                                                     | /thing
+                                                           |
+$.accountNum [equalTo] 1234                                | Warning: JSON path expression '$.accountNum' failed to<<<<< Body does not match
```

Comment:
I feel like here, on the Request side it should just say `(empty)` rather than rendering an error message. We shouldn't really treat it as an error as when scanning a whole set of stubs matching a request with an empty body it's perfectly legit to test it against a stub that requires one.

Reply:
OK - I've reworked it so the diff rendered outputs no body if there is no body, and the body itself if the expression does not match, which seems to be the standard way.